charts/datadog/values.yaml

```diff
@@ -254,6 +254,19 @@ datadog:
   ## ref: https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments#environment-variables
   checksCardinality:  # low, orchestrator or high (not set by default to avoid overriding existing DD_CHECKS_TAG_CARDINALITY configurations, the default value in the Agent is low)
 
+  # SBOM configuration
+  sbom:
+    # datadog.sbom.enabled -- Enable SBOM (Software Bill Of Materials) collection
```

Comment:
it can be good to document that it require Agent 7.46.0 and it is still a beta feature